cmd/autocomplete/collaborators_add.go

```diff
@@ -25,15 +27,25 @@ func CollaboratorsAddAutoComplete(c *cli.Context) error {
 		return nil
 	}
 
+	var wg sync.WaitGroup
+	wg.Add(len(apps))
 	setEmails := make(map[string]bool)
 	for _, app := range apps {
-		appCollaborators, err := api.CollaboratorsList(app.Name)
-		if err != nil {
-			return nil
-		}
-		for _, col := range appCollaborators {
-			setEmails[col.Email] = true
-		}
+		go func(app *api.App, setEmails map[string]bool) {
+			appCollaborators, erro := api.CollaboratorsList(app.Name)
+			if erro != nil {
+				err = erro
+				return
+			}
+			for _, col := range appCollaborators {
+				setEmails[col.Email] = true
```

Comment:
map in go are not "concurrency safe", you need to set a mutex to lock it or to use channels (which the the recommanded way)